From 0863bd240872c538825c0e262621372c452bb4e2 Mon Sep 17 00:00:00 2001
From: Daniel van Vugt <daniel.van.vugt@canonical.com>
Date: Fri, 6 Sep 2019 17:49:48 +0800
Subject: [PATCH 18/30] output-kms: Use portable formatting for 64-bit integers

On 64-bit systems uint64_t is %lu but on 32-bit systems it is %llu.

https://gitlab.gnome.org/GNOME/mutter/merge_requests/778
---
 src/backends/native/meta-output-kms.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/backends/native/meta-output-kms.c b/src/backends/native/meta-output-kms.c
index 1ec17b9f9..73e4e4c94 100644
--- a/src/backends/native/meta-output-kms.c
+++ b/src/backends/native/meta-output-kms.c
@@ -70,7 +70,7 @@ meta_output_kms_set_underscan (MetaOutput    *output,
       hborder = MIN (128, (uint64_t) round (crtc->current_mode->width * 0.05));
       vborder = MIN (128, (uint64_t) round (crtc->current_mode->height * 0.05));
 
-      g_debug ("Setting underscan of connector %s to %lu x %lu",
+      g_debug ("Setting underscan of connector %s to %" G_GUINT64_FORMAT " x %" G_GUINT64_FORMAT,
                meta_kms_connector_get_name (output_kms->kms_connector),
                hborder, vborder);
 
@@ -104,7 +104,7 @@ meta_output_kms_set_power_save_mode (MetaOutput    *output,
 {
   MetaOutputKms *output_kms = output->driver_private;
 
-  g_debug ("Setting DPMS state of connector %s to %lu",
+  g_debug ("Setting DPMS state of connector %s to %" G_GUINT64_FORMAT,
            meta_kms_connector_get_name (output_kms->kms_connector),
            dpms_state);
 
-- 
2.23.0

